test/parallel/test-mime-api.js

```diff
@@ -158,3 +160,8 @@ assert.throws(() => params.set(`x${NOT_HTTP_TOKEN_CODE_POINT}`, 'x'), /parameter
 assert.throws(() => params.set('x', `${NOT_HTTP_QUOTED_STRING_CODE_POINT};`), /parameter value/i);
 assert.throws(() => params.set('x', `${NOT_HTTP_QUOTED_STRING_CODE_POINT}x`), /parameter value/i);
 assert.throws(() => params.set('x', `x${NOT_HTTP_QUOTED_STRING_CODE_POINT}`), /parameter value/i);
+
+assert.strictEqual(toASCIILower('someThing'), 'something');
+assert.strictEqual(toASCIILower('SomeThing'), 'something');
+assert.strictEqual(toASCIILower('SomeThing3'), 'something3');
+assert.strictEqual(toASCIILower('ABCDEFGHIJKLMNOPQRSTUVWXYZ'), 'ABCDEFGHIJKLMNOPQRSTUVWXYZ'.toLowerCase());
```

Comment:
```suggestion
assert.strictEqual(toASCIILower('ABCDEFGHIJKLMNOPQRSTUVWXYZ'), 'abcdefghijklmnopqrstuvwxyz');
```

Reply:
Any reason you didn't apply this suggestion? No strong feelings, just curious.

Reply:
I think i would have applied it. Seems that i missed applying it. Should we apply it... i mean ci cd takes about 5 hours to complete. Its your call. If you want apply it. I would have discussed if I did not wanted to apply it ;).